src/main/java/net/hypixel/nerdbot/util/gson/HypixelPlayerResponse.java

```diff
@@ -0,0 +1,151 @@
+        @Getter private String currentClickEffect;
+        @Getter private String currentGadget;
+        @SerializedName("claimed_potato_talisman")
+        @Getter private Instant claimedPotatoTalisman;
+        @SerializedName("skyblock_free_cookie")
+        @Getter private Instant skyblockFreeCookie;
+        @SerializedName("claimed_century_cake")
+        @Getter private Instant claimedCenturyCake;
+        @SerializedName("scorpius_bribe_120")
+        @Getter private Instant scorpiusBribe120;
+        @Getter private Map<String, Long> voting;
+        @Getter private Map<String, Integer> petConsumables;
+        @Getter private Map<String, Integer> achievements;
+        @Getter private Map<String, Instant> achievementRewardsNew;
+
+        // Rank
+        private String packageRank;
+        private String newPackageRank;
+        private String monthlyPackageRank;
+        private String rank;
+        private String prefix;
+        private String monthlyRankColor;
+        private String rankPlusColor;
+        private String mostRecentMonthlyPackageRank;
+
+        // Stats (Only SkyBlock Currently)
+        @Getter private Stats stats;
+
+        public List<String> getAchievementsOneTime() {
+            if (this.achievementsOneTimeFixed == null)
+                this.achievementsOneTimeFixed = this.achievementsOneTime.stream()
+                    .filter(obj -> (obj instanceof String))
+                    .map(String::valueOf)
+                    .collect(Collectors.toList());
+
+            return achievementsOneTimeFixed;
+        }
+
+        public UUID getUniqueId() {
+            return Util.toUUID(this.uuid);
+        }
+
+    }
+
+    public static class SocialMedia {
+
+        private boolean prompt;
+        @Getter private Map<Service, String> links = new HashMap<>();
+
+        public enum Service {
+
+            TWITTER,
+            YOUTUBE,
+            INSTAGRAM,
+            TWITCH,
+            DISCORD,
+            HYPIXEL
+
+        }
+
+    }
+
+    public static class Stats {
+
+        @SerializedName("SkyBlock")
+        @Getter private SkyBlock skyBlock;
+
+        public static class SkyBlock {
+
+            private Map<String, Profile> profiles = new HashMap<>();
+
+            public Optional<Set<Profile>> getProfiles() {
+                return this.profiles == null ? Optional.empty() : Optional.of(new HashSet<>(this.profiles.values()));
+            }
+
+            public static class Profile {
+
+                @SerializedName("profile_id")
+                private String profileId;
+                @SerializedName("cute_name")
+                private String cuteName;
+
+                public String getCuteName() {
+                    return this.cuteName;
+                }
+
+                public UUID getUniqueId() {
+                    return Util.toUUID(this.profileId);
+                }
+
+            }
+
+        }
+
+    }
+
+}
```

Comment:
Get rid of all these blank lines

Reply:
why it makes it easier to know where one block ends compared with another

Reply:
Isn't that the point of indentations lol